README.md

```diff
@@ -1049,6 +1049,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Musab Habeeb](https://github.com/Musab1258)
   - [Musibau Anifowose Bolaji](https://github.com/adigson)
   - [Muthu Annamalai V.](https://github.com/muthuannamalai12)
+  - [Muskan Singh](https://github.com/Radiosnack)
```

Comment:
Please add your name in line 1051, after Musibau Anifowose Bolaji.
Let us know if you need any help doing that! 🙂